### PR TITLE
Added productBoxLayout config for manufacturer page

### PR DESCRIPTION
**Pull request by shyim**
## Description
Please describe your pull request:
* Why is it necessary?
Adds productBoxLayout config to shop configuration
* What does it improve?
Makes easy to change product layout of manufacturer pages
* Does it have side effects?
Nope

| Questions        | Answers
| ---------------- | -------------------------------------------------------
| BC breaks?       | no
| Tests pass?      | yes
| Related tickets? | If this PR fixes an existing [Issue](https://issues.shopware.com) ticket, please add its complete issue URL.
| How to test?     | Please describe how to best verify that this PR is correct.

### CODE REVIEW

File: _sql/migrations/793-add-productbox-for-manufacturer.php [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+class Migrations_Migration793 Extends Shopware\Components\Migrations\AbstractMigration
+{
+    public function up($modus)
+    {
+        $sql = <<<'EOD'
+INSERT IGNORE INTO `s_core_config_elements` (`form_id`, `name`, `value`, `label`, `description`, `type`, `required`, `position`, `scope`) VALUES
+(144, 'manufacturerProductBoxLayout', 's:5:"basic";', 'Hersteller-Seite Produkt Layout', 'Mit Hilfe des Produkt Layouts können Sie entscheiden, wie Ihre Produkte auf der Hersteller-Seite dargestellt werden sollen. Wählen Sie eines der drei unterschiedlichen Layouts um die Ansicht perfekt auf Ihr Produktsortiment abzustimmen.', 'product-box-layout-select', 0, 0, 1);
+
+SET @elementId = (SELECT `id` FROM `s_core_config_elements` WHERE `form_id`= 144 AND `name`="manufacturerProductBoxLayout" LIMIT 1);
+
```
Comment (hlohaus) [inline _sql/migrations/793-add-productbox-for-manufacturer.php]: wir machen normalerweise für jedes SQL Statement ein:
 $this->addSql($sql);
:thumbsup:

Comment (shyim) [reply _sql/migrations/793-add-productbox-for-manufacturer.php]: @hlohaus ist aktualisiert :)

**Comment by janbuecker** [comment]
Thanks @shyim 

We decided to close this PR because of time constraints and limited overall benefit to the product. If this change is still important to you, please create a new issue on https://issues.shopware.com so people can up-vote this feature.

**Comment by shyim** [comment]
Created https://issues.shopware.com/issues/SW-17483

**Comment by shyim** [comment]
@janbuecker how many upvotes are required to reopen :)

**Comment by shyim** [comment]
The issue has now 18 upvotes. Are their a interest to reopen this pr?

**Pull request closed without merging**